x-pack/plugins/upgrade_assistant/server/lib/es_migration_apis.ts

```diff
@@ -20,6 +25,31 @@ export async function getUpgradeAssistantStatus(
   const cluster = getClusterDeprecations(deprecations, isCloudEnabled);
   const indices = getCombinedIndexInfos(deprecations);
 
+  const indexNames = indices.map(({ index }) => index!);
+
+  // If we have found deprecation information for index/indices, add some additional information;
+  // is it currently open or closed?
+  if (indexNames.length) {
+    // According to https://www.elastic.co/guide/en/elasticsearch/reference/7.6/cluster-state.html
```

Comment:
Is it worth having a quick conversation with an ES engineer to confirm this is the correct approach?

Reply:
I spoke with @DaveCTurner about this and we concluded that `_cluster/state` is probably the best place to get information about closed indices.

The addition of https://github.com/jloleysens/kibana/blob/d73316a2bb5e9d0436ba9d324a678e02b344cb6c/x-pack/test/upgrade_assistant_integration/upgrade_assistant/status.ts#L28

Should keep us on top of any changes that may happen to that response.